lib/ui/example/client/provider.js

```diff
@@ -78,25 +78,62 @@ export default class ReactProvider extends Provider {
     this.api = api;
     this.api.setOptions({
       name: 'REACT-STORYBOOK',
+      sortStoriesByKind: true,
```

Comment:
As far as I checked - no. But maybe it's already has another meaning.. Anyway the hierarchy is built after the filtering:
![image](https://user-images.githubusercontent.com/7867954/27475511-29b8a7ae-580e-11e7-83e1-87b2b95ec5a0.png)